.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: '💡 Feature Request'
+about: Submit your idea for a change in the codebase.
+---
+
+# Feature Request
+
+<!--
+  This issue should serve for you to present or pitch an idea to the maintainers - but remember that it would be better if you were to submit a PR instead 🤗
+-->
+
```

Comment:
Can we add a section for "Possible implementation". It forces people to think about how the feature would work, rather than just saying "it should do X".

Reply:
I can refactor the code sample section to be a "sub-section" of a possible implementation. Probably code sample isn't generic enough